tools/changestate.php

```diff
@@ -39,6 +39,13 @@
     fatal_error( _("You are not permitted to perform this action.") );
 }
 
+if ($transition->is_disabled($project))
+{
+    $body = _("This function is disabled while the project is in the Smooth Reading Pool.<br/><br/>If you believe this is an error, please contact db-req for assistance.");
```

Comment:
This text should take `why_disabled()` into account.

Reply:
For translation purposes and HTML rendering, I would do:
```
$body = '<p>' . _("This function is disabled...") . '</p>' .
        '<p>' . _("If you believe this...") . '</p>';
```
Also note that we use HTML5, not XHTML, so it would be `<br>` not `<br/>` if you were to use them.